### ISSUE TITLE
Adding terms to psi-mod-CV to link modifications with amino acids

### ISSUE THREAD

**Issue by pmt706** [HUPO-PSI/psi-mod-CV#11]
Would everyone be ok if I were to run through the psi-mod CV and put in appropriate links to the amino acids modified?   For instance, right now, MOD:01356 is_a MOD:00916 (modified serine), when I would categorize it instead as MOD:01356 is_a MOD:00002 (O-glycosyl serine, which itself is_a MOD:00916).  There are many such circumstances in the MOD:01157 (protein modification characterized by amino acid modified) side of the CV that need addressing.

**Comment by pabinz** [comment]
In that respect, if we want to reorganise the strict relationships of terms, which makes perfect sense, yes, there are many instances.
MOD:01356 would have
is_a MOD:00002 !  O-glycosyl-L-serine                                                (instead of MOD:00916)
is_a MOD:00517 ! Hex1HexNAc1NeuAc1 glycosylated residue


And I would keep the order of the is_a  by starting with the proximal childhood of MOD:01157 followed by the proximal childhood of MOD:01156  (although this priorization has no impact on the relationship itself. Only a human reading convenience)

Cheers



De : pmt706 <notifications@github.com>
Envoyé : lundi 10 février 2020 20:43
À : HUPO-PSI/psi-mod-CV <psi-mod-CV@noreply.github.com>
Cc : Subscribed <subscribed@noreply.github.com>
Objet : [HUPO-PSI/psi-mod-CV] Adding terms to psi-mod-CV to link modifications with amino acids (#11)


Would everyone be ok if I were to run through the psi-mod CV and put in appropriate links to the amino acids modified? For instance, right now, MOD:01356 is_a MOD:00916 (modified serine), when I would categorize it instead as MOD:01356 is_a MOD:00002 (O-glycosyl serine, which itself is_a MOD:00916). There are many such circumstances in the MOD:01157 (protein modification characterized by amino acid modified) side of the CV that need addressing.

**Comment by pmt706** [comment]
Agreed!  My goal is to connect these leaf nodes to as few parents as possible and to root them ultimately to MOD:01157 and MOD:01156 (and to MOD:00538 where necessary)... also to pick up any missing links to UniMod/RESID if they exist.

**Comment by edeutsch** [comment]
Although I didn't realize we had this problem, in that I thought it was already very nicely hierarchical, if you see ways to make the parent-child relationships better, I am all for it! Agreed!
